### PR TITLE
Fix numerous typos.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: libethcore/Farm.h [hunk 1]
```diff
@@ -415,7 +415,7 @@ class Farm: public FarmFace
 	 * @brief Called from a Miner to note a WorkPackage has a solution.
 	 * @param _p The solution.
 	 * @param _wp The WorkPackage that the Solution is for.
-	 * @return true iff the solution was good (implying that mining should be .
+	 * @return true if the solution was good (implying that mining should be .
```
Comment (chfast) [inline libethcore/Farm.h]: Iff actually means "if and only if".

**Comment by XhmikosR** [comment]
Oh, all right I'll revert these 2 tomorrow.

File: libethash-cuda/CUDAMiner.cpp [hunk 2]
```diff
@@ -384,7 +384,7 @@ bool CUDAMiner::cuda_init(
 		if(dagSize128 != m_dag_size || !dag)
 		{
 			// create mining buffers
-			cudalog << "Generating mining buffers"; //TODO whats up with this?
```
Comment (MariusVanDerWijden) [inline libethash-cuda/CUDAMiner.cpp]: Could you delete this whole comment (I pushed it years ago on accident)

Comment (XhmikosR) [reply libethash-cuda/CUDAMiner.cpp]: Done

**Pull request merged**